tests/modules/ebi-metagenomics/checkm2/checkm2/test.yml

```diff
@@ -0,0 +1,35 @@
+# TODO: fix test_checkm2 test
+# Doesn't work because of download db issue
+# OSError: [Errno 30] Read-only file system: '/usr/local/lib/python3.8/site-packages/checkm2/version/diamond_path.json'
+
+- name: "checkm2_stub"
```

Comment:
Should this not have the same name as the test in `tests/modules/ebi-metagenomics/checkm2/checkm2/main.nf`, so `"test_checkm2"`?